lib/modules/platform/gitea/index.ts

```diff
@@ -491,9 +498,10 @@ const platform: Platform = {
     branchName,
     prTitle: title,
     state = 'all',
+    includeOtherAuthors,
   }: FindPRConfig): Promise<Pr | null> {
     logger.debug(`findPr(${branchName}, ${title!}, ${state})`);
-    const prList = await platform.getPrList();
+    const prList = await platform.getPrList(includeOtherAuthors);
```

Comment:
I think this is incorrect. The pr list from getPrList() is computed only once and returned from cache every other time. Instead the PR list should be filtered to also include branches ending in `reconfigure`